src/overlay.js

```diff
@@ -312,6 +312,14 @@
             this.placement  = location instanceof $.Point ?
                 placement :
                 $.OverlayPlacement.TOP_LEFT;
+        },
+        /**
+         * @function
+         * @returns {OpenSeadragon.Rect} overlay bounds
+         */
+        getBounds: function() {
+
+            return this.bounds;
```

Comment:
This should be `return this.bounds.clone();` so we're returning a copy that the app can modify; otherwise they might accidentally change our real bounds unexpectedly.